build/package/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM brandoshmando/go:1.13.4-ticket-lock AS binary
```

Comment:
> build/package/Dockerfile

I'd prefer not to have a `build/` directory.. let's just keep this in the root. And, in fact, I had to move the Dockerfile to root in order to get it to build properly (as otherwise it was unable to see the `cmd` directory).

Reply:
Agreed, build dir is probably overkill right now when we're building this exclusively on a linux machine. But someday, when zseries hits it big and we're building on several OS'...

You should be able to get it to build if you set the context with `.` :

`docker build -f build/package/Dockerfile .`